frontend/src/pages/Admin/ModifyCoursePage.jsx

```
@@ -111,88 +111,104 @@ const ModifyCoursePage = () => {
+              Description
+            </label>
+            <textarea
+              id="description"
+              name="description"
+              value={course.description}
+              onChange={handleChange}
+              className="w-full p-3 border border-gray-300 rounded-xl focus:ring-2 focus:ring-blue-400"
+              placeholder="Enter course description"
+              rows="4"
+              required
+            ></textarea>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)